docs/unquoted_attributes_examples.cpp

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
Please remove this file from docs, instead, you can create a new html file that demostrates the unquoted attributes at fixtures/html/...